backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -48,7 +44,5 @@ trait BackendLifecycleActorFactory {
   lazy val fileHashingFunction: Option[FileHashingFunction] = None
   lazy val fileHashingWorkerCount: Int = 50
 
-  final lazy val fileContentsHasherActor: ActorRef = actorSystem.actorOf(RoundRobinPool(fileHashingWorkerCount)
-    .props(FileHasherWorkerActor.props(fileHashingFunction)),
-    "FileContentsHasherActor")
+  def fileContentsHasherActorProps: Props = FileHasherWorkerActor.props(fileHashingFunction)
```

Comment:
On the chance that the ticket I'm going to mention isn't what sparked this, there's #1377